esti/lakectl_util.go

```diff
@@ -41,9 +41,14 @@ func lakectlLocation() string {
 }
 
 func LakectlWithParams(accessKeyID, secretAccessKey, endPointURL string) string {
+	return LakectlWithParamsWithPosixPerms(accessKeyID, secretAccessKey, endPointURL, false)
+}
+
+func LakectlWithParamsWithPosixPerms(accessKeyID, secretAccessKey, endPointURL string, withPosixPerms bool) string {
```

Comment:
Adding this explicitly - 
Rather keep this easy to delete (assuming it'll happen in the near future),
Than mess the whole code with changes (of adding a new param).